### PR TITLE
Dashboard Font Managment (Part 1)

### PR DESCRIPTION
**Pull request by niwinz**

### CODE REVIEW

File: frontend/src/app/main/data/dashboard/fonts.cljs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+;;   (ptk/reify ::add-font
+;;     ptk/UpdateEvent
+;;     (update [_ state]
+;;       (let [id   (uuid/next)
+;;             font (-> font
+;;                      (assoc :created-at (dt/now))
+;;                      (assoc :id id)
+;;                      (assoc :status :draft))]
+;;         (js/console.log (clj->js font))
+;;         (assoc-in state [:dashboard-fonts id] font)))))
```
Comment (hirunatan) [inline frontend/src/app/main/data/dashboard/fonts.cljs:94]: may be deleted?

File: frontend/src/app/main/ui/dashboard/fonts.cljs [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+     [:nav
+      [:ul
+       [:li {:class (when (= section :fonts) "active")}
+        [:a {:on-click go-fonts} (tr "labels.custom-fonts")]]
+       [:li {:class (when (= section :providers) "active")}
+        [:a {:on-click go-providers} (tr "labels.font-providers")]]]]
+
+     [:div]]))
+
+(defn- prepare-fonts
```
Comment (hirunatan) [inline frontend/src/app/main/ui/dashboard/fonts.cljs:74]: Maybe move this to some data/ or even util/?

File: frontend/translations/es.po [hunk 3]
```diff
@@ -11,6 +11,68 @@ msgstr ""
...
+
+msgid "labels.font-providers"
+msgstr "Proveedores de fuentes"
+
+msgid "labels.upload-custom-fonts"
+msgstr "Subir fuente"
+
+#, markdown
+msgid "dashboard.fonts.hero-text1"
+msgstr "Any web font you upload here will be added to the font family list available at the text properties of the files of this team. Fonts with the same font family name will be grouped as a **single font family**. You can upload fonts with the following formats: **TTF, OTF and WOFF** (only one will be needed)."
```
Comment (hirunatan) [inline frontend/translations/es.po:61]: Translate this?

File: frontend/translations/es.po [hunk 4]
```diff
@@ -11,6 +11,68 @@ msgstr ""
...
+
+#, markdown
+msgid "dashboard.fonts.hero-text2"
+msgstr "You should only upload fonts you own or have license to use in Penpot. Find out more in the Content rights section of [Penpot's Terms of Service](https://penpot.app/terms.html). You also might want to read about [font licensing](2)."
```
Comment (hirunatan) [inline frontend/translations/es.po:65]: Same

File: frontend/src/app/main/data/dashboard/fonts.cljs (hunk 1 above)
Comment (niwinz) [reply frontend/src/app/main/data/dashboard/fonts.cljs:94]: sure

File: frontend/src/app/main/ui/dashboard/fonts.cljs (hunk 2 above)
Comment (niwinz) [reply frontend/src/app/main/ui/dashboard/fonts.cljs:74]: Sure, but there are a lot of work pending on this, so I will do it in the next PR

File: frontend/translations/es.po (hunk 3 above)
Comment (niwinz) [reply frontend/translations/es.po:61]: Yes, this is pending because I need to talk with UX team about this text.

**Pull request merged**